src/agent/src/agent.cpp

```diff
@@ -11,11 +12,13 @@
 #include <thread>
 
 Agent::Agent(const std::string& configFile, std::unique_ptr<ISignalHandler> signalHandler)
-    : m_agentInfo([this]() { return m_sysInfo.os(); }, [this]() { return m_sysInfo.networks(); })
-    , m_messageQueue(std::make_shared<MultiTypeQueue>())
-    , m_signalHandler(std::move(signalHandler))
-    , m_configurationParser(configFile.empty() ? configuration::ConfigurationParser()
+    : m_configurationParser(configFile.empty() ? configuration::ConfigurationParser()
                                                : configuration::ConfigurationParser(std::filesystem::path(configFile)))
+    , m_dataPath(m_configurationParser.GetConfig<std::string>("agent", "path.data").value_or(config::DEFAULT_DATA_PATH))
```

Comment:
Don't you see necessary to add `path.data` to the default configuration file?
(`etc/config/wazuh-agent.yml`)